check_sites.py

```diff
@@ -111,10 +115,17 @@ def main():
                 return
         old_checker = RwsCheck(old_sites, etlds, icanns)
         check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), rws_checker.load_sets())
-        # TODO: add variable and check for subtracted_sets in case of user 
-        # removing old set from the list
     else:
         check_sets = rws_checker.load_sets()
+        if testing_sets:
+            temp_sets = {}
+            for testing_set in testing_sets:
+                if testing_set not in check_sets:
+                    error_texts.append("There was an error loading the set:\n" + testing_set +
+                        " could not be found in related_website_sets.JSON")
```

Comment:
I think it might be better to print an error and quit entirely here, since the CLI arg was invalid. I'd rather force people to get the CLI arg completely right, than to have them see *some* of the output and (incorrectly) assume everything was fine.

Reply:
As I understand, you're saying in the case where a user enters multiple primaries and one of them is not in the RWS list, we should exit without giving error information about the sets whose primaries they entered correctly? If so, I think I disagree - I'd prefer we give us much error information as possible when they run the command. My guess is that most users are only going to be running the command with a single primary argument, so this would be moot anyway.

Reply:
Agree that it's probably rare that people will supply more than one site.

My fear about showing as much error information as possible in this case is that I think it would be easy to miss this error, in the midst of others. But I guess if there are other errors, then the dev will still know that there's something they need to fix; so if everything else is fine (and there is no other output), then it'll be easy to spot this error, anyway. So I'm ok with making this error non-fatal.